src/darkmode/darkmode.ts

```diff
@@ -0,0 +1,77 @@
+export function toggleDarkMode(enable: boolean) {
+  if (enable) {
+    applyDarkMode();
+  } else {
+    removeDarkMode();
+  }
+}
+
+export function applyDarkMode() {
+  const darkModeStyle = `
```

Comment:
I know the CSS isn't ideal because it uses the class names which are sort of dynamic, but I'm hoping just for a Loom demo or whatever that it'll work? (only on the homepage obviously)